### PR TITLE
Add ClubMember program

### PR DESCRIPTION
**Pull request by sarafuentes3025**

### CODE REVIEW

File: src/main/java/com/sfuentes/ClubMember.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    for (double clave : weightPerScale.get(1)) {
+      sum += clave;
+    }
+
+    average = sum / weightPerScale.get(1).size();
+
+    if (average > previousWeight)
+      comparison.put("Gained weight", average - previousWeight);
+    else if (average < previousWeight)
+      comparison.put("Under weight", previousWeight - average);
```
Comment (aandraca) [inline src/main/java/com/sfuentes/ClubMember.java:33]: Under is a preposition it cannot be used in this case.
I think "lose weight" would be better

File: src/main/java/com/sfuentes/ClubMember.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    this.previousWeight = previousWeight;
+    this.weightPerScale = weightPerScale;
+  }
+
+  public Map<String, Double> getWeightComparison(double previousWeight, Map<Integer, List<Double>> weightPerScale) {
+    Map<String, Double> comparison = new HashMap<>();
+    double sum = 0.0;
+    double average;
+
+    for (double clave : weightPerScale.get(1)) {
```
Comment (aandraca) [inline src/main/java/com/sfuentes/ClubMember.java:24]: is the parameter weightPerScale the same as the property? the one you set in the constructor?
if that's the case you shouldn't ask for it in this method, just use the property.

File: src/main/java/com/sfuentes/ClubMember.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ClubMember {
+  int memberID;
+  Map<Integer, List<Double>> weightPerScale;
+  double previousWeight;
+
+  public ClubMember(int memberID, double previousWeight, Map<Integer, List<Double>> weightPerScale) {
```
Comment (aandraca) [inline src/main/java/com/sfuentes/ClubMember.java:12]: What is the purpose of memberId? I see in your code is always 1, but not sure how it works.

**Review by aandraca — CHANGES_REQUESTED** [review]

File: src/main/java/com/sfuentes/ClubMember.java (hunk 1 above)
Comment (sarafuentes3025) [reply src/main/java/com/sfuentes/ClubMember.java]: fixed

File: src/main/java/com/sfuentes/ClubMember.java (hunk 2 above)
Comment (sarafuentes3025) [reply src/main/java/com/sfuentes/ClubMember.java]: fixed

**Review by aandraca — APPROVED** [review]

**Pull request merged**